starlite/datastructures/response_containers.py

```diff
@@ -24,21 +24,24 @@
 
 from pydantic import BaseConfig, FilePath, validator
 from pydantic.generics import GenericModel
-from starlette.responses import FileResponse, RedirectResponse
-from starlette.responses import Response as StarletteResponse
-from starlette.responses import StreamingResponse
+from typing_extensions import Literal
 
 from starlite.datastructures.background_tasks import BackgroundTask, BackgroundTasks
 from starlite.datastructures.cookie import Cookie
 from starlite.enums import MediaType
 from starlite.exceptions import ImproperlyConfiguredException
-from starlite.response import TemplateResponse
+from starlite.response import (
+    FileResponse,
+    RedirectResponse,
+    StreamingResponse,
+    TemplateResponse,
+)
 
 if TYPE_CHECKING:
     from starlite.app import Starlite
     from starlite.connection import Request
 
-R = TypeVar("R", bound=StarletteResponse)
+R = TypeVar("R")
```

Comment:
I agree

Reply:
I bound it to `Response` - before i had some cyclical import issues, but thats fixed now.

Reply:
Ok, I remembered why I did this. It shouldn't be a protocol - it should be `Response`, but mypy is going ape shit for some reason. I have no clue why to be honest, i tried fixing it but its some issue with GenericModel + Generic and mypy in this setup. Anyhow, for now I will keep it without binding, because we are using ResponseContainer generics internally.